bundles/binding/org.openhab.binding.ihc/src/main/java/org/openhab/binding/ihc/internal/IhcGenericBindingProvider.java

```diff
@@ -87,50 +94,92 @@ public void processBindingConfiguration(String context, Item item,
 
 		IhcBindingConfig config = new IhcBindingConfig();
 		config.itemType = item.getClass();
+		config.outBindingOnly = true;
+		config.outCommandMap = new HashMap<Command, IhcOutCommandConfig>();
 		
-		String[] configParts = bindingConfig.trim().split(":");
-
-		if (configParts.length > 2) {
-			throw new BindingConfigParseException(
-					"IHC / ELKO LS binding must contain of max two two parts separated by ':'");
-		}
-
-		if (bindingConfig.startsWith(">")) {
-
-			if (configParts.length == 1) {
-				config.outBindingOnly = true;
-				String resourceId = configParts[0].replace(">", "");
-
-				if (resourceId.startsWith("0x")) {
-					config.resourceId = Integer.parseInt(
-							resourceId.replace("0x", ""), 16);
-				} else {
-					config.resourceId = Integer.parseInt(resourceId);
+		String[] splittedCommands = bindingConfig.split(",");
+		
+		for(String split : splittedCommands) {
+			if (split.startsWith(">")) {
+				try {
+					//out binding
+					String resourceCommand = split.substring(1);
+					
+					final String commandRegex = "\\[([\\w*]+):(0x[0-9a-fA-F]+|\\d+)(?::(\\d+)){0,1}\\]"; 
+					Pattern p = Pattern.compile(commandRegex);
+					Matcher m = p.matcher(resourceCommand);
+	
+					IhcOutCommandConfig outConfig = new IhcOutCommandConfig();
+					
+					if(!m.matches()) {
+						if(splittedCommands.length < 2) {
+							//assume old style out command
+							outConfig.command = null; //wildcard
+							outConfig.resourceId = getResourceIdFromString(resourceCommand);
+												
+						} else {
+							throw new BindingConfigParseException(
+									"Item '"
+											+ item.getName()
+											+ "' has invalid out binding config");
+						}
+					} else {
+						//new style out binding
+						Command command = TypeParser.parseCommand(
+								item.getAcceptedCommandTypes(), m.group(1));
+	
+						if(command == null && m.group(0) != "*") {
+							throw new BindingConfigParseException(
+									"Item '"
+											+ item.getName()
+											+ " invalid Command: "+ m.group(1));
+						}
+						
+						outConfig.command = command;
+						outConfig.resourceId = getResourceIdFromString(m.group(2));
+						
+						if(m.groupCount() == 3 && m.group(3) != null) {
+							outConfig.value = Integer.parseInt(m.group(3));
+						}
+					}
+					
+					config.outCommandMap.put(outConfig.command, outConfig);
+				}
+				catch(Exception e) {
+					logger.warn("Error in output config for item: "+ item.getName(), e);
 				}
-				
-			} else {
-				throw new BindingConfigParseException(
-						"When configuration start with '>', refresh interval is not supported ");
-			}
-
-		} else {
-
-			String resourceId = configParts[0];
-
-			if (resourceId.startsWith("0x")) {
-				config.resourceId = Integer.parseInt(
-						resourceId.replace("0x", ""), 16);
 			} else {
-				config.resourceId = Integer.parseInt(resourceId);
+				if(splittedCommands.length < 2) {
+					String[] configParts = bindingConfig.trim().split(":");
```

Comment:
Length of the configParts should be checked. If length is bigger than 2, exception should be thrown. In old code this was done in line 93.